roles/telemetry_autoscaling/tasks/configure_heat.yml

```diff
@@ -27,19 +27,48 @@
 
   # TODO: Pass the parameters correctly. When I tried to pass the parameters
   # into the template.yaml file, they weren't passed to the instance initially.
-- name: RHOSO-12657 Configure heat template for automatically scaling instances
-  ansible.builtin.template:
-    src: instance.yaml.j2
-    dest: ~/templates/autoscaling/vnf/instance.yaml
-
-- name: RHOSO-12658 Create the resource to reference in the heat template
-  ansible.builtin.copy:
-    dest: ~/templates/autoscaling/vnf/resources.yaml
-    content: |
-      resource_registry:
-        "OS::Nova::Server::VNF": ./instance.yaml
-
-- name: RHOSO-12659 Create the deployment template for heat to control instance scaling
-  ansible.builtin.template:
-    dest: ~/templates/autoscaling/vnf/template.yaml
-    src: template.yaml.j2
+
+- name: Congigure deployment template
+  block:
+    - name: RHOSO-12657 Configure heat template for automatically scaling instances
+      ansible.builtin.template:
+        src: instance.yaml.j2
+        dest: ~/templates/autoscaling/vnf/instance.yaml
+
+    - name: RHOSO-12658 Create the resource to reference in the heat template
+      ansible.builtin.copy:
+        dest: ~/templates/autoscaling/vnf/resources.yaml
+        content: |
+          resource_registry:
+            "OS::Nova::Server::VNF": ./instance.yaml
+
+    - name: RHOSO-12659 Create the deployment template for heat to control instance scaling
+      ansible.builtin.template:
+        dest: ~/templates/autoscaling/vnf/template.yaml
+        src: template.yaml.j2
+
+    - name: Add pass to test results
+      delegate_to: localhost
+      lineinfile:
+        path: "{{ logs_dir }}/test_run_result.xml"
+        line: '<testcase classname="openstack-observability"  name="configure-hest-template-automatically-scaling-instances"  time="1"/>'
+        insertbefore: "</testsuite>"
```

Comment:
This is the only test result file I can see.

I need to look for the one generated by the junit plugin